### PR TITLE
fix(ui5-tabcontainer): bottom align tab strip item text with no additional text

### PR DESCRIPTION
**Pull request by kskondov**
Strip tabs text is bottom aligned even when there is no additional text
Fixes: #6764

### CODE REVIEW

File: packages/main/src/TabInStrip.hbs [hunk 1]
```diff
@@ -64,5 +64,8 @@
 {{#*inline additionalText}}
 	{{#if this.additionalText}}
 		<span class="ui5-tab-strip-itemAdditionalText" id="{{this._id}}-additionalText">{{this.additionalText}}</span>
+
+	{{else if this.icon}}
+		<span class="ui5-tab-strip-itemAdditionalText-hidden" id="{{this._id}}-additionalText-hidden"></span>
```
Comment (gmkv) [inline packages/main/src/TabInStrip.hbs:69]: Refactor this to use a getter `classes` in Tab.ts like it is used on line 2. The object this getter will return would be accessed with `stripClasses.additionalText` which will contain the appropriate class `ui5-tab-strip-itemAdditionalText`, and include `ui5-tab-strip-itemAdditionalText--hidden`. Combine it with the existing getter `stripClasses` but change the type of the return value.

File: packages/main/src/themes/TabInStrip.css [hunk 2]
```diff
@@ -229,6 +229,11 @@
 	display: flex;
 }
 
+.ui5-tab-strip-itemAdditionalText-hidden {
+	visibility: hidden ;
```
Comment (gmkv) [inline packages/main/src/themes/TabInStrip.css:233]: remove space before semicolon

**Review by gmkv — CHANGES_REQUESTED** [review]

File: packages/main/src/TabInStrip.hbs (hunk 1 above)
Comment (kskondov) [reply packages/main/src/TabInStrip.hbs]: Done

File: packages/main/src/themes/TabInStrip.css [hunk 3]
```diff
@@ -258,7 +258,7 @@
 	padding: 0 0.188rem;
 }
 
-.ui5-tab-strip-item--textOnly.ui5-tab-strip-item--withAddionalText {
```
Comment (gmkv) [inline packages/main/src/themes/TabInStrip.css:256]: There is one more place where `withAddionalText` is mispelled in the code. Could you fix it there too?

**Review by gmkv — CHANGES_REQUESTED** [review]

**Review by gmkv — APPROVED** [review]

**Pull request merged**